### PR TITLE
Bug Fix

### PR DESCRIPTION
**Pull request by Ismail-AD**
@aritra-tech Do review code and merge as soon as possible :)

### CODE REVIEW

**Comment by aritra-tech** [comment]
Please fix the `ktlint` checks

**Comment by Ismail-AD** [comment]
checked !!! successful at my side

File: app/build.gradle.kts [hunk 1]
```diff
@@ -115,6 +115,7 @@ dependencies {
 
     // Coil
     implementation(libs.coil.compose)
+    implementation(libs.richeditor.compose)
```
Comment (aritra-tech) [inline app/build.gradle.kts:118]: What is the use of the dependency in this PR?
Please remove it @Ismail-AD

**Review by aritra-tech — CHANGES_REQUESTED** [review]

File: app/build.gradle.kts (hunk 1 above)
Comment (Ismail-AD) [reply app/build.gradle.kts]: @aritra-tech it is done  do review and merge

**Comment by aritra-tech** [comment]
Are you sure that the code that you have done is working?
When I tested it I couldn't save empty notes on giving `spaces` also the save button doesn't appear.
When I tested your branch there were some changes that are not in your branch.
Please check it out once.

**Comment by Ismail-AD** [comment]
I don't get it it the logic I have implemented so far will not allow not to be get saved until there will be no characters and buttons to save will remain disappeared and will only appear once user will type some characters rather spaces and my branch should do that I will double check ✅ IF MY ABOVE logic is sufficient or  I am wrong at some point ?

**Comment by aritra-tech** [comment]
This issue that you have talked about was already handled previously.
I wanted the user can also save empty notes or let's say he/she doesn't give the title and writes only the notes then it will be saved. Just like Google Keep Notes

**Comment by Ismail-AD** [comment]
I don't get it it the logic I have implemented so far will not allow not to be get saved until there will be no characters and buttons to save will remain disappeared and will only appear once user will type some characters rather spaces and my branch should do that I will double check ✅ IF MY ABOVE logic is sufficient or  I am wrong at some point ?

**Comment by aritra-tech** [comment]
> You mean if title is even not written then note with only description should be saved and if I am wrong provide a demo VIDEO for change implementation

Yeah right.

**Comment by Ismail-AD** [comment]
**IS THIS GOOD TO GO**  @aritra-tech 
WhatsApp Image 2023-10-23 at 6 49 05 PM

**Comment by aritra-tech** [comment]
> **IS THIS GOOD TO GO** @aritra-tech WhatsApp Image 2023-10-23 at 6 49 05 PM

Yeah right.

**Comment by Ismail-AD** [comment]
@aritra-tech now what :) review & merge it

**Comment by aritra-tech** [comment]
Working fine but when I run your branch I couldn't find many changes which are already present on `develop`.
Do one thing.
- Right Click on `develop` and click on `Checkout`.
- After checkout to develop update the develop branch.
- Once `develop` branch is updated then checkout to your branch after checkout click on the develop branch once and you will see an option named `Merge develop into bugFix` on doing that your `bugFix` branch will be updated and you can see the changes then only.
Please do that.

**Comment by Ismail-AD** [comment]
@aritra-tech actually i have to do these on github or at android studio as on github on right click web options are displayed not any checkout any video you can provide for this or should i open a new PR ?

**Comment by aritra-tech** [comment]
I would suggest opening a new pr.
Before creating a branch from develop first pull the develop branch and then create a branch from develop

**Comment by aritra-tech** [comment]
Any update @Ismail-AD

**Comment by Ismail-AD** [comment]
Involved in preparation of exams so you can assign it to someone else

**Comment by aritra-tech** [comment]
> Involved in preparation of exams so you can assign it to someone else

It's okay @Ismail-AD
All the best for your exams

**Pull request closed without merging**